windows/Clipboard/Clipboard.vcxproj

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props')" />

```

Comment:
Not for now, but I didn't realize how coupled community modules were to host details. E.g. this means we will break community modules every stimewe bump a NuGet package.

I think there's a cppwinrt way to do this, but there are still places not using cppwinrt in RNW. 

I feel like we need to rework community module build logic if we don't want them to frequently break.

Reply:
yes, we mention this in the docs that the cppwinrt version must match.